assets/tests/components/shuttleMapPage.test.tsx

```diff
@@ -158,6 +159,29 @@ describe("Shuttle Map Page", () => {
     )
     expect(result.asFragment()).toMatchSnapshot()
   })
+
+  test("clicking a shuttle on the map dispatches select event", async () => {
+    const label = "clickMe"
+    ;(useShuttleVehicles as jest.Mock).mockImplementationOnce(() => [
+      { ...shuttle, label: label },
+    ])
+    const mockDispatch = jest.fn()
+    const result = render(
+      <StateDispatchProvider
+        state={{ ...initialState, selectedShuttleRunIds: "all" }}
+        dispatch={mockDispatch}
+      >
+        <BrowserRouter>
+          <ShuttleMapPage />
+        </BrowserRouter>
+      </StateDispatchProvider>
+    )
+
+    await userEvent.click(result.getByText(label))
```

Comment:
**nitpick:** I thought we had been moving towards making queries against the `screen` object rather than using `result` from `render`?

Reply:
Thank you! This habit is hard to break 😅